docs/en/02_Developer_Guides/09_Security/04_Secure_Coding.md

```diff
@@ -762,11 +762,15 @@ disable this behaviour using `CanonicalURLMiddleware::singleton()->setForceBasic
 configuration in YAML.
 
 We also want to ensure cookies are not shared between secure and non-secure sessions, so we must tell Silverstripe CMS to 
-use a [secure session](https://docs.silverstripe.org/en/3/developer_guides/cookies_and_sessions/sessions/#secure-session-cookie). 
-To do this, you may set the `cookie_secure` parameter to `true` in your `config.yml` for `Session`
+use a [secure session](https://docs.silverstripe.org/en/3/developer_guides/cookies_and_sessions/sessions/#secure-session-cookie).
```

Comment:
```suggestion
use a [secure session](https://docs.silverstripe.org/en/4/developer_guides/cookies_and_sessions/sessions/#secure-session-cookie).
```

Probably also seeing if there are other en/3/ links around, if so please open a new issue with corresponding PR to find and replace them

Reply:
I'll swap this to be a relative URL instead. This probably was carried over when the `4` branch was created and we want to avoid having to change it again in the next major release docs.

Reply:
I have checked and the only other link to v3 docs is intentional - but I've created an issue to swap all the absolute URLs to relative URLs so we don't have this issue in future majors.